static/app/views/projectInstall/createProject.tsx

```diff
@@ -300,33 +296,6 @@ class CreateProject extends Component<Props, State> {
     }
   };
 
-  trackIssueAlertOptionSelectedEvent(
-    projectData: Project,
-    isDefaultRules: boolean | undefined,
-    shouldCreateCustomRule: boolean | undefined,
-    ruleId: string | undefined
-  ) {
-    const {organization} = this.props;
-
-    let data: RuleEventData = {
-      eventKey: 'new_project.alert_rule_selected',
```

Comment:
@Neo-Zhixing are we dropping the `new_project.alert_rule_selected` event?

Reply:
Yes, let's unify the metric alert & issue alert analytic events

Reply:
@Neo-Zhixing hmmm ok, but it seems like we are missing some info like the project id that we used to have